docs/topics/proc_selecting-the-jdk-builder-image-for-your-application.adoc

```diff
@@ -0,0 +1,57 @@
+[id='selecting-the-jdk-builder-image-for-your-application_{context}']
+= Selecting a JDK image for your {runtime} application
+Edit your application `pom.xml` file to specify the image for building and running your {runtime} application.
+
+.Prerequisites
+* A `pom.xml` file for your application.
```

Comment:
This prerequisite is a little vague. Do we want to say that the user should have a pom.xml ready before they start this task?
Consider rephrasing to something like: The `pom.xml` file is created.

Reply:
Fixed. Thanks for the suggestion @inoxx03.